shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -90,36 +81,66 @@ private static void printBlockMasterInfo(String spaces) {
               BlockMasterInfoField.FREE_BYTES, BlockMasterInfoField.CAPACITY_BYTES_ON_TIERS,
               BlockMasterInfoField.USED_BYTES_ON_TIERS));
       BlockMasterInfo blockMasterInfo = client.getBlockMasterInfo(blockMasterInfoFilter);
+      if (blockMasterInfo == null) {
+        throw new IOException("Cannot get block master info from block master client");
+      }
 
-      System.out.println(spaces + "Live workers: " + blockMasterInfo.getLiveWorkerNum());
-      System.out.println(spaces + "Lost workers: " + blockMasterInfo.getLostWorkerNum());
+      System.out.println(spaces + "Live Workers: " + blockMasterInfo.getLiveWorkerNum());
+      System.out.println(spaces + "Lost Workers: " + blockMasterInfo.getLostWorkerNum());
 
       System.out.println(spaces + "Total Capacity: "
           + FormatUtils.getSizeFromBytes(blockMasterInfo.getCapacityBytes()));
       Map<String, Long> totalCapacityOnTiers = blockMasterInfo.getCapacityBytesOnTiers();
-      for (Map.Entry<String, Long> capacityBytesTier : totalCapacityOnTiers.entrySet()) {
-        long value = capacityBytesTier.getValue();
-        if (value > 0) {
-          System.out.println(spaces + spaces + "Tier: " + capacityBytesTier.getKey()
-              + "  Size: " + FormatUtils.getSizeFromBytes(value));
+      if (totalCapacityOnTiers != null) {
```

Comment:
when would this be null?

Reply:
when Thrift side does not successfully providing this info, all other fields will just print out 0 or null values, but for (Map.Entry<String, Long> capacityBytesTier : totalCapacityOnTiers.entrySet()) will have problem running because totalCapacityOnTiers is null.